### DISCUSSION TITLE
Cannot find ble-com-setup script

### DISCUSSION THREAD

**Discussion by szabomFB** [Jakeler/ble-serial#25] [Q&A] [answered]
I am trying to setup ble-serial for windows, but I cannot find the executable to run for the com0com setup part. The command is not recognized in terminal, and I tried looking through folders in C:\Python38\Lib\site-packages\ble_serial but could not find any file named ble-com-setup.exe. Any advice would be appreciated!

**Comment by Jakeler** [comment, marked as answer]
Usually it is in `C:\Users\USERNAME\AppData\Local\Programs\Python\Python38\Scripts\ble-com-setup.exe` or the corresponding scripts path if installed somewhere else. 
But I had a mistake in this release: it used the Linux package for Windows as well, which does not the include `ble-com-setup` script... Should be fixed now with 2.0.1, please do a `pip uninstall ble-serial` + `pip install ble-serial` again and let me know if it works.

**Reply by szabomFB** [reply]
Thank you for pointing me in the right direction, I was indeed able to find it in the scripts folder and run as expected!
